README.md

```diff
@@ -199,6 +199,8 @@ Testing base64 encoding/decoding of the large blob into the newly allocated buff
 
 Testing parsing and simple calculating of values from a big JSON file.
 
+D implementations except Mir (Ion and Asdf) have inaccurate number parsing.
```

Comment:
I think it's better to just fix it, please don't hesitate to change the existing code.

Reply:
Mir is the only D library that have correct number parsing. Other libraries use standard D library, which is inaccurate. I can’t fix them, because this would mean they would need to depend on Mir. My proposal of adding Mir as dependency to standard library has been rejected. So, I think I did what I can for this case. The precise number parsing is quite complex and computationally expensive task. If library uses inaccurate number parsing then it has a bug that gives unfair advantage in benchmarks. As far as I know simdjson and the Rust libraries are precise. Seems like D is the only language that doesn’t have accurate number parsing in the standard library.

Reply:
I see, thanks for the clarification. Could you elaborate on this in the text though? At least some links (ideally the bugreports) and maybe an example could be helpful.

Reply:
Interesting, thank you. You may be interested in my other project that verifies the parsing and binary representation of the float numbers (that's my playground for the internal protobuf-like communication protocol I'm using at my work): https://gitlab.com/nuald-grp/floats-binary-representation If you want, you can add D sample, and we can verify the code across different implementations.

Reply:
Cool, btw, you may be interested in [Ryu](https://github.com/ulfjack/ryu) for the project, it generates the shortest precise decimal representations. Mir and many other languages or libs support it. MS is going to add it to its C++ library.